docs/style-guide.rst

```diff
@@ -5,6 +5,8 @@ This :doc:`style guide </guide/writing/style-guides>` specifies how we write on
 It's just a start,
 but everything has to start somewhere :)
 
+For other style guides, see :doc:`Style Guides </guide/writing/style-guides>`.
```

Comment:
I might suggest using `.. seealso::` for these blocks, so give it a bit more definition, and nicer semantics. 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso